### PR TITLE
Feature: Add p-values to tidy.lmodel2()

### PR DESCRIPTION
**Pull request by mattle24**
Addressing https://github.com/tidymodels/broom/issues/570

### CODE REVIEW

**Comment by mattle24** [comment]
On the first build AppVeyor failed installing curl:

`Error: (converted from warning) cannot remove prior installation of package 'curl'`

**Comment by alexpghayes** [comment]
Can you update the documentation to include that the p-value is one-tailed (?) and calculated via a permutation test (?), and possibly note why this is?

**Comment by mattle24** [comment]
Sure!

**Comment by mattle24** [comment]
I updated the documentation to note the one-tailed p-values and explained why they're calculated with permutation tests. After looking through the `lmodel2` documentation I'm still not sure why they're one-sided though.

File: R/lmodel2-tidiers.R [hunk 1]
```diff
@@ -55,6 +61,7 @@ tidy.lmodel2 <- function(x, ...) {
 
   ret %>%
     inner_join(confints, by = c("method", "term")) %>% 
+    select(-p.value, dplyr::everything()) %>%
```
Comment (alexpghayes) [inline R/lmodel2-tidiers.R]: Why do you need the `select()` line here? What does it do?

File: R/stats-lm-tidiers.R [hunk 2]
```diff
@@ -279,5 +279,52 @@ process_lm <- function(ret, x, conf.int = FALSE, conf.level = .95,
...
+  process_lm(ret, x, conf.int = conf.int, conf.level = conf.level, 
+             exponentiate = exponentiate)
+}
+
+#' TODO: document
+#' @rdname tidy.lm.beta
+#' @export
+tidy.summary.lm.beta <- function (x, ...) {
+  co <- stats::coef(x)
+  nn <- c("estimate", 'standardized.estimate', "std.error", "statistic", "p.value")
```
Comment (alexpghayes) [inline R/stats-lm-tidiers.R]: Let's use `std.estimate` instead of `standardized.estimate` for the column name.

File: R/stats-lm-tidiers.R [hunk 3]
```diff
@@ -279,5 +279,52 @@ process_lm <- function(ret, x, conf.int = FALSE, conf.level = .95,
...
+#' CarRegS = lm.beta::lm.beta(CarReg)
+#' 
+#' # Table format
+#' tidy(CarRegS)
+#'
+#' @export
+#' @family lm tidiers
+tidy.lm.beta <- function (x, conf.int = FALSE, conf.level = 0.95, exponentiate = FALSE, 
+                          quick = FALSE, ...) {
+  if (quick) {
```
Comment (alexpghayes) [inline R/stats-lm-tidiers.R]: This looks like a copy-paste of `tidy.lm()`, which is a very reasonable place to start. However, `tidy.lm()` is a mess and is way more complicated than it needs to be. Would you mind attempting something more self-contained in the vein of `tidy.nls()` for example (https://github.com/tidymodels/broom/blob/master/R/stats-nls-tidiers.R#L32)?

**Comment by alexpghayes** [comment]
The `lmodel2` stuff looks great. Left some comments on the `lm.beta` tidiers!

File: R/lmodel2-tidiers.R (hunk 1 above)
Comment (mattle24) [reply R/lmodel2-tidiers.R]: This changes the order of the columns so p.value is the rightmost column.

**Comment by mattle24** [comment]
Sorry about all of this. I think I had an old and bad commit leftover on my forked repo's master. I pulled instead of reseting to the tidymodels/master, and that caused a bunch of problem. To fix this, I did some hard resets, so things may look a little funky now. I took out the lm.beta things though, all that should be here is the lmodel2 stuff.

**Comment by alexpghayes** [comment]
Looks good, thanks!

**Pull request merged**